guillotina/contrib/dbusers/__init__.py

```diff
@@ -0,0 +1,31 @@
+from guillotina import configure
+from guillotina.i18n import MessageFactory
+
+_ = MessageFactory("guillotina.contrib.dbusers")
+
+
+app_settings = {
+    "auth_user_identifiers": [
+        "guillotina.contrib.dbusers.users.DBUserIdentifier"
+    ]
+}
+
+configure.permission("guillotina.NotAuthenticated", "")
+configure.permission("guillotina.Authenticated", "")
+configure.permission("guillotina.AddUser", title="Add plone user")
+configure.permission("guillotina.AddGroup", title="Add plone group")
+configure.grant(permission="guillotina.AccessContent", role="guillotina.Anonymous")
```

Comment:
isn't that kinda dangerous?

Reply:
hmmm, yah, I wonder why that is there.

Reply:
it seems it has been there from the beginning: https://github.com/plone/guillotina_dbusers/blame/739e0f9ada206402077894f62684f033f1200f7a/guillotina_dbusers/__init__.py#L18

Reply:
I think I got to the root of the problem.
I fixed it with https://github.com/plone/guillotina/pull/566/commits/8e488e9b2886cc018bc084ebba0bf2534d884207